### PR TITLE
[pos restaurant] handle the failure response to display a proper error message

### PR DESCRIPTION
**Pull request by dka-odoo**
Task :-  https://www.odoo.com/web#id=21739&view_type=form&model=project.task&action=327&menu_id=4720

pad :-  https://pad.odoo.com/p/r.azIMx66SDNlYuBpV

### CODE REVIEW

File: addons/pos_restaurant/static/src/js/floors.js [hunk 1]
```diff
@@ -292,6 +292,13 @@ var TableWidget = PosBaseWidget.extend({
             }
             floorplan.update_toolbar();
             self.destroy();
+        }, function(err, event) {
+            self.gui.show_popup('error', {
+                'title':_t('Changes could not be saved'),
+                'body': _t('Check your internet connection and access rights'),
```
Comment (bst-odoo) [inline addons/pos_restaurant/static/src/js/floors.js]: Change body string : "You must be connected to the internet to save your changes."

Comment (dka-odoo) [inline addons/pos_restaurant/static/src/js/floors.js]: well, what if the user has no sufficient rights?  in that case,  we would have the same failure response.

Comment (bst-odoo) [inline addons/pos_restaurant/static/src/js/floors.js]: In this case, the user shouldn't even be able to edit the table plan. The "pen" icon shouldn't be visible.

Comment (bst-odoo) [inline addons/pos_restaurant/static/src/js/floors.js]: I've updated the spec according to your remark. thanks for the feedback.

Comment (dka-odoo) [inline addons/pos_restaurant/static/src/js/floors.js]: alright!

**Pull request merged**